src/base/configuration.cpp

```diff
@@ -110,8 +110,96 @@ std::string Configuration::getVersionExtra() {
+       << "you want to license CVC4 under this license, please configure CVC4\n"
+       << "with the \"--bsd\" option before building from sources.\n\n";
+  else
+    ss << "CVC4 is open-source and is covered by the BSD license (modified)."
+       << "\n\n";
+
+  ss << "THIS SOFTWARE IS PROVIDED AS-IS, WITHOUT ANY WARRANTIES.\n"
+     << "USE AT YOUR OWN RISK.\n\n";
+ 
+  ss << "This version of CVC4 is linked against the following third party "
+     << "libraries:\n\n";
+
+  ss << "  ANTLR - Another Tool for Language Recognition\n"
+     << "  http://www.antlr.org\n"
+     << "  Copyright (c) 2012-2017 The ANTLR Project."
+     << "\n\n";
+
+  if (Configuration::isBuiltWithGmp())
+    ss << "  GMP - Gnu Multi Precision Arithmetic Library\n"
+       << "  http://gmplib.org\n"
+       << "  Copyright (c) 1991, 1993-2016 Free Software Foundation, Inc."
+       << "\n\n";
+
+  if (Configuration::isBuiltWithCln())
+    ss << "  CLN - Class Library for Numbers\n"
+       << "  http://www.ginac.de/CLN\n"
+       << "  Copyright (c) Bruno Haible 1995-2008.\n"
+       << "  Copyright (c) Richard B. Kreckel 2000-2014.\n"
+       << "  Copyright (c) Alexei Sheplyakov 2008, 2010."
+       << "\n\n";
+
+  if (Configuration::isBuiltWithGlpk())
+    ss << "  glpk-cut-log -  a modified version of GPLK, "
+       << "the GNU Linear Programming Kit\n"
+       << "  http://github.com/timothy-king/glpk-cut-log\n"
+       << "  Copyright (c) 2000-2012 Andrew Makhorin, Department for Applied "
+       << "Informatics,\n"
+       << "  Moscow Aviation Institute, Moscow, Russia.\n"
+       << "  Copyright (c) 2013-2014 Tim King, New York University."
+       << "\n\n";
+
+  if (Configuration::isBuiltWithAbc())
+    ss << "  ABC - A System for Sequentiyl Synthesis and Verification\n"
+       << "  https//bitbucket.org/alanmi/abc\n"
+       << "  Copyright (c) The Regents of the University of California."
+       << "\n\n";
+
+  if (Configuration::isBuiltWithCryptominisat())
+    ss << "  CryptoMiniSat - An Advanced SAT Solver\n"
+       << "  http://github.com/msoos/cryptominisat\n"
+       << "  Copyright (c) Mate Soos.\n\n";
+
+  if (Configuration::isBuiltWithReadline())
+    ss << "  GNU Readline\n"
+       << "  http://cnswww.cns.cwru.edu/php/chet/readline/rltop.html\n"
+       << "  Copyright (c) 1989-2016 Free Software Foundation, Inc."
+       << "\n\n";
+
+  if (Configuration::isBuiltWithLfsc())
+    ss << "  LFSC Proof Checker\n"
+       << "  http://github.com/CVC4/LFSC\n"
+       << "  Copyright (c) 2012, 2013 The University of Iowa."
+       << "\n\n";
+
+  ss << "See the file COPYING (distributed with the source code, and with\n"
+     << "all binaries) for the full CVC4 copyright, licensing, and (lack of)\n"
+     << "warranty information.\n";
+  return ss.str();
+}
+
 std::string Configuration::about() {
-  return CVC4_ABOUT_STRING;
+  std::stringstream ss;
+  ss << "This is CVC4 version " << CVC4_RELEASE_STRING;
+  if (Configuration::isGitBuild())
```

Comment:
I prefer parens even around single-line if statements. It is quite hard to read because some stuff is indented but not part of an if statement (e.g. line 200). It can also prevent [bugs](https://www.imperialviolet.org/2014/02/22/applebug.html).